ssl/record/record_locl.h

```diff
@@ -31,6 +31,8 @@
 #define RECORD_LAYER_reset_empty_record_count(rl) \
                                                 ((rl)->empty_record_count = 0)
 #define RECORD_LAYER_get_empty_record_count(rl) ((rl)->empty_record_count)
+#define RECORD_LAYER_is_first_record(rl)        ((rl)->is_first_record)
+#define RECORD_LAYER_set_first_record(rl, val)  ((rl)->is_first_record = (val))
```

Comment:
If is_first_record is really just a boolean, you may not need the vull versatility of setting a specific vlaue; just a RECORD_LAYER_clear_first_record() call would be sufficient.